src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -153,8 +186,43 @@ def create(self, name, host, network):
                                        keyring=keyring,
                                        extra_config={'config': extra_config})
 
+    def _check_safe_to_destroy(self, mon_id):
+        # type: (str) -> None
+        ret, out, err = self.mgr.check_mon_command({
+            'prefix': 'quorum_status',
+        })
+        try:
+            j = json.loads(out)
+        except Exception as e:
+            raise OrchestratorError('failed to parse quorum status')
+
+        mons = [m['name'] for m in j['monmap']['mons']]
+        if mon_id not in mons:
+            logger.info('Safe to remove mon.%s: not in monmap (%s)' % (
+                mon_id, mons))
+            return
+        new_mons = [m for m in mons if m != mon_id]
+        new_quorum = [m for m in j['quorum_names'] if m != mon_id]
+        if len(new_quorum) > len(new_mons) / 2:
+            logger.info('Safe to remove mon.%s: new quorum should be %s (from %s)' % (mon_id, new_quorum, new_mons))
```

Comment:
Since this is a logger.info, then it is better to right?
```
logger.info('Safe to remove mon.%s: new quorum should be %s (from %s)', mon_id, new_quorum, new_mons)
```
Otherwise, why not keep the same style using `f' '` strings?